bundle/config/mutator/resourcemutator/apply_bundle_permissions.go

```diff
@@ -0,0 +1,207 @@
+package resourcemutator
```

Comment:
This comes from bundle/permissions/mutator.go ?

Why is move not detected? If there are changes, can you describe them in PR description?

Reply:
There are no changes, I moved private methods from utils.go into a single file